tests/testproject/TestProject.AppHost/TestProject.AppHost.csproj

```diff
@@ -13,6 +13,7 @@
     <AspireProjectOrPackageReference Include="Aspire.Hosting.Azure.CosmosDB" />
     <AspireProjectOrPackageReference Include="Aspire.Hosting.Azure.EventHubs" />
     <AspireProjectOrPackageReference Include="Aspire.Hosting.AppHost" />
+    <AspireProjectOrPackageReference Include="Aspire.Hosting.Milvus" />
```

Comment:
As part of the work for https://github.com/dotnet/aspire/issues/4294, we are now doing these functional, end-to-end tests in the Aspire.Hosting.X.Tests functional tests. So you can remove these changes to the `testproject` and Aspire.Hosting.Tests. We don't need to test all the integrations in our core tests anymore.

Reply:
Will do, I saw redis and a postgres still in there so wasn't sure.

Reply:
Yeah - my plan is to only have a core set of extensions (< 5) in the end-to-end test. The test is still good, we just don't need it to grow indefinitely.